discv5/discv5-wire.md

```diff
@@ -154,6 +154,10 @@ handshake packet.
 
 ![handshake packet layout](./img/handshake-packet-layout.png)
 
+### Notification Packet (`flag = 3`)
+
+Aside from the flag value, this container is identical to a [message packet](#ordinary-message-packet).
```

Comment:
The most important thing about this new packet type, is that the recipient should not trigger a handshake if it cannot be decrypted. We need to mention this at least once.

Reply:
I also think that 'notifications' should be for response messages PONG, NODES, TALKRESP.

Reply:
> I also think that 'notifications' should be for response messages PONG, NODES, TALKRESP.

This does make sense theoretically, but I don't see the point to break the working wire protocol here. Furthermore, I like the logic separation between Messages = request-response relationship and notifications = notifications.

Reply:
I made the changes you wanted. A better name for the notification packet is "SessionMsg", indicating that this message assumes an established session (and otherwise is dropped), and then on Protocol message level, the messages are either of type request, response or notification. What do you think?

Reply:
Nonetheless, whoareyou and handshake are responses on their level (TLS level), so the name "Notification" is intuitive since it doesn't trigger any kind of response.